config/initializers/sentry.rb

```diff
@@ -1,6 +1,19 @@
 return unless ENV['SENTRY_DSN'].present?
 
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = ENV['SENTRY_DSN']
-  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+
+
+  filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters.map(&:to_s))
```

Comment:
This is supposed to replace the `sanitize_fields` method, which has been removed. I am unsure whether the `to_s` is still right here, advice welcome!

Reply:
Have checked with the ParameterFilter documentation and it seems the to_s won't hurt. It could potentially be removed as I think the filter will auto-cast it, but it's not a problem to have it here.